GVRf/Framework/src/org/gearvrf/debug/ShellCommandHandler.java

```diff
@@ -42,25 +43,31 @@ public ShellCommandHandler(GVRContext gvrContext) {
     }
 
     @Command
-    public void lua() {
-        enterLanguage(GVRScriptManager.LANG_LUA);
+    public Object lua() {
+        return enterLanguage(GVRScriptManager.LANG_LUA);
     }
 
     @Command
-    public void js() {
-        enterLanguage(GVRScriptManager.LANG_JAVASCRIPT);
+    public Object js() {
+        return enterLanguage(GVRScriptManager.LANG_JAVASCRIPT);
     }
 
-    private void enterLanguage(String language) {
+    private Object enterLanguage(String language) {
         ScriptEngine engine = mGVRContext.getScriptManager().getEngine(language);
 
         if (engine == null) {
-            mShell.getOutputConverter().convertOutput("Cannot find the language engine for " + language);
-            return;
+            return "Cannot find the language engine for " + language;
         }
 
         mScriptHandler = new ScriptHandler(mGVRContext, language, engine);
         mShell.setLineProcessor(mScriptHandler);
+
+        return null;
```

Comment:
you changed the return type to Object, but only return non-null when the engine is null.  correct?  this is just for outputting an error?